src/expandableCalendar/weekCalendar.js

```diff
@@ -30,7 +30,8 @@ class WeekCalendar extends Component {
     /** whether to have shadow/elevation for the calendar */
     allowShadow: PropTypes.bool,
     /** whether to hide the names of the week days */
-    hideDayNames: PropTypes.bool
+    hideDayNames: PropTypes.bool,
+    theme: PropTypes.object,
```

Comment:
No need to declare 'theme' prop, it is declared in Calendar prop types. CalendarList gets all Calendar props and WeekCalendar gets all CalendarList props.